web/src/components/molecules/Asset/Asset/AssetBody/index.tsx

```diff
@@ -108,18 +108,23 @@ const AssetBody: React.FC<Props> = ({
             <UnzipFileList style={{ minHeight: "400px" }}></UnzipFileList>
           </Card>
         )}
-        <DownloadButton type="ghost" filename={fileName} url={url} displayDefaultIcon={true} />
+        <DownloadButton
+          type="ghost"
+          filename={asset?.fileName}
+          url={url}
+          displayDefaultIcon={true}
```

Comment:
Just an FYI, I think with boolean props you can just write `displayDefaultIcon` without the `={true}` part

Reply:
Don't need to change unless you want to.